cms/djangoapps/contentstore/views/course.py

```diff
@@ -502,7 +502,10 @@ def course_listing(request):
     org = request.GET.get('org', '') if optimization_enabled else None
     courses_iter, in_process_course_actions = get_courses_accessible_to_user(request, org)
     user = request.user
+    # remove this temporary log after getting results
+    start_time = time.time()
     libraries = _accessible_libraries_iter(request.user, org) if LIBRARIES_ENABLED else []
+    log.info("_accessible_libraries_iter completed in [%f]", (time.time() - start_time))
```

Comment:
Are we planning to merge this logging as well? If so, please file a ticket for eventual removal

Reply:
Yes, just created [EDUCATOR-1732](https://openedx.atlassian.net/browse/EDUCATOR-1732) for log removal.